src/main/java/io/passport/server/model/LoginResponse.java

```diff
@@ -0,0 +1,17 @@
+package io.passport.server.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.keycloak.representations.AccessTokenResponse;
+
+/**
+ * Response class for login request.
+ */
+@Getter
+@Setter
+public class LoginResponse {
+
+    private AccessTokenResponse authResponse;
+
+    private String userId;
```

Comment:
Instead of creating a LoginResponse class, we can add userId to access token and extract it from the token whenever necessary.